packages/artillery/lib/platform/aws-ecs/legacy/run-cluster.js

```diff
@@ -654,6 +658,15 @@ async function tryRunCluster(scriptPath, options, artilleryReporter) {
         await launchLeadTask(context);
       }
 
+      setCloudwatchRetentionOnInterval(
```

Comment:
nitpick but whan does "OnInterval" mean here? if it refers to the implementation detail of using `setInterval` then imo it should be just `setCloudWatchRetention`